engine/app/components/citizens_advice_components/input.rb

```diff
@@ -4,7 +4,7 @@ module CitizensAdviceComponents
   class Input < Base
     attr_reader :name, :label, :error_message, :hint, :value
 
-    def initialize(name:, label:, type:, options: nil)
+    def initialize(name:, label:, type:, value: nil, options: nil)
```

Comment:
Given this lVar is now no longer used you can drop this too. But as it's symbolised it won't flag if you don't (Linting might show it up though).